components/Players/PlayerManager.vue

```diff
@@ -195,6 +203,34 @@ export default Vue.extend({
         default:
           return this.getCurrentItem.Name;
       }
+    },
+    timeLeft(): number {
```

Comment:
Similarly to the item duration, this should be a getter, since we're bound to re-use it at some point (Example: if we want to add a preference to show remaining time instead of total duration, like some players do).